### PR TITLE
Fixed Dashboard buttons

### PR DESCRIPTION
**Pull request by quiz3**
Fixed Dashboard buttons, removed QUIT, and tidied up the use case and view model.

REVIEW REQUIRED: `DashboardView` Lines 130-136. Is it true that those are not necessary given the action listener actions already specified?

### CODE REVIEW

File: src/main/java/view/DashboardView.java [hunk 1]
```diff
@@ -94,46 +109,29 @@ public void actionPerformed(ActionEvent evt) {
...
+        // TODO: Display user stats, once they are available.
 
         // TODO: Display table of information about stocks owned by the user.
 
-        // Pay attention to the input from the buttons in this view.
-        buy.addActionListener(this);
-        sell.addActionListener(this);
-        news.addActionListener(this);
-        reset.addActionListener(this);
-        quit.addActionListener(this);
```
Comment (gursi26) [inline src/main/java/view/DashboardView.java:135]: Not necessary

**Review by gursi26 — CHANGES_REQUESTED** [review]

File: src/main/java/view/DashboardView.java (hunk 1 above)
Comment (quiz3) [reply src/main/java/view/DashboardView.java:135]: I agree, I thought I had already removed those lines.

Comment (quiz3) [reply src/main/java/view/DashboardView.java:135]: Wait do you mean it is not necessary to remove the lines?

**Comment by quiz3** [comment]
Okay, now that we settled the issue in person, approve the changes please.

**Review by gursi26 — APPROVED** [review]

**Pull request merged**